src/modules/dashboard/components/SubscribedColoniesList/SubscribedColoniesDropdown.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import ColonyAvatar from '~core/ColonyAvatar';
+import DropdownMenu, {
+  DropdownMenuSection,
+  DropdownMenuItem,
+} from '~core/DropdownMenu';
+import Popover from '~core/Popover';
+import { ProcessedColony } from '~data/generated';
+import styles from './SubscribedColoniesList.css';
+
+const displayName =
+  'dashboard.SubscribedColoniesList.SubscribedColoniesDropdown';
+
+export type Colony = Pick<
+  ProcessedColony,
+  | 'colonyName'
+  | 'colonyAddress'
+  | 'id'
+  | 'displayName'
+  | 'avatarHash'
+  | 'avatarURL'
+>;
+interface Props {
+  activeColony: Colony;
+  coloniesList: Colony[];
+}
+
+const SubscribedColoniesDropdown = ({ activeColony, coloniesList }: Props) => {
+  return (
+    <Popover
+      content={() => {
+        return (
+          <DropdownMenu>
```

Comment:
If you have a really long number of colonies, it will break the view / dropdown.

See example below:

![mobile-view-more-colonies-break-view](https://user-images.githubusercontent.com/1193222/175383202-5e077ca3-54ab-4556-982f-94d13d88accf.gif)

The dropdown should have a fixed height _(max-height)_ after which it should just allow scrolling between the items

Reply:
Less colonies work fine though

![mobile-view-less-colonies-ok](https://user-images.githubusercontent.com/1193222/175383588-3eb67b22-7a1f-44ab-9df4-740c2eba3858.gif)